apps/web/modules/survey/templates/lib/organization.ts

```diff
@@ -0,0 +1,36 @@
+import { Organization, Prisma } from "@prisma/client";
+import { cache as reactCache } from "react";
+import { prisma } from "@formbricks/database";
+import { cache } from "@formbricks/lib/cache";
+import { organizationCache } from "@formbricks/lib/organization/cache";
+import { DatabaseError } from "@formbricks/types/errors";
+
+export const getOrganizationAIKeys = reactCache(
```

Comment:
This service is defined at 4 different places in the survey module, pls move this to `/modules/survey/lib/organization.ts`, please take a last look at any more such service that you can find and move them 🙏